### PR TITLE
Update centralized-deployment-of-add-ins.md

### PR DESCRIPTION
**Pull request by rokeates**

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/manage/centralized-deployment-of-add-ins.md [hunk 1]
```diff
@@ -62,20 +46,41 @@ Centralized Deployment doesn't support the following:
...
+- For Word, Excel, and PowerPoint add-ins, your users must be using one of the following:
+  - On a Windows device, Version 1704 or later of Office 365 ProPlus.
+  - On a Mac, Version 15.34 or later.
+- For Outlook, your users must be using one of the following: 
+  - Version 1701 or later of Office 365 ProPlus.
+  - Version 1808 or later of Office Professional Plus 2019 or Office Standard 2019.
+  - Version 16.0.4494.1000 or later of Office Professional Plus 2016 (MSI) or Office Standard 2016 (MSI)\*
+  - Version 15.0.4937.1000 of later of Office Professional Plus 2013 (MSI) or Office Standard 2013 (MSI)\*
+    
+    \*In MSI versions of Outlook, admin-installed add-ins will show in the appropriate ribbon in Outlook but doesn't show the add-in in 'My add-ins' section.
```
Comment (nenonix) [inline Office365-Admin/manage/centralized-deployment-of-add-ins.md]: *In MSI versions of Outlook, admin-installed add-ins will show in the appropriate ribbon in Outlook but doesn't show the add-in in 'My add-ins' section.

This does not make sense. Suggest (if this is what is meant):
MSI versions of Outlook show admin-installed add-ins in the appropriate Outlook ribbon, not the "My add-ins" section.

File: Office365-Admin/manage/centralized-deployment-of-add-ins.md [hunk 2]
```diff
@@ -25,30 +25,14 @@ description: "Determine if your Office 365 tenant and users meet the requirement
 # Determine if Centralized Deployment of add-ins works for your organization
 
 Centralized Deployment is the recommended and most feature-rich way for most customers to deploy Office add-ins to users and groups within your Office 365 organization. If you're an Office 365 admin, use this guidance to determine if your tenant and users meet the requirements so that you can use Centralized Deployment.
-  
+Centralized Deployment supports Windows, Mac and Office Online Clients.
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/centralized-deployment-of-add-ins.md]: Centralized **deployment** supports Windows, Mac and Office Online Clients.

Please change to lower case d.

File: Office365-Admin/manage/centralized-deployment-of-add-ins.md [hunk 3]
```diff
@@ -62,20 +46,41 @@ Centralized Deployment doesn't support the following:
...
+### Office Requirements
+
+- For Word, Excel, and PowerPoint add-ins, your users must be using one of the following:
+  - On a Windows device, Version 1704 or later of Office 365 ProPlus.
+  - On a Mac, Version 15.34 or later.
+- For Outlook, your users must be using one of the following: 
+  - Version 1701 or later of Office 365 ProPlus.
+  - Version 1808 or later of Office Professional Plus 2019 or Office Standard 2019.
+  - Version 16.0.4494.1000 or later of Office Professional Plus 2016 (MSI) or Office Standard 2016 (MSI)\*
+  - Version 15.0.4937.1000 of later of Office Professional Plus 2013 (MSI) or Office Standard 2013 (MSI)\*
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/centralized-deployment-of-add-ins.md]: Version 15.0.4937.1000 **or** later of Office Professional Plus 2013 (MSI) or Office Standard 2013 (MSI)\*

Please change (or).

File: Office365-Admin/manage/centralized-deployment-of-add-ins.md [hunk 4]
```diff
@@ -62,20 +46,41 @@ Centralized Deployment doesn't support the following:
...
+    
+    \*In MSI versions of Outlook, admin-installed add-ins will show in the appropriate ribbon in Outlook but doesn't show the add-in in 'My add-ins' section.
     
-### Client requirements
 
-The following table shows the clients that currently support the Centralized Deployment feature.
+#### Find out if Office 365 ProPlus is installed
+
+To use Office 365 ProPlus, a user must have an Office 365 account and been assigned a license. For more information, see [Overview of Office 365 ProPlus](https://go.microsoft.com/fwlink/p/?linkid=846328)..
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/centralized-deployment-of-add-ins.md]: To use Office 365 ProPlus, a user must have an Office 365 account and **have** been assigned a license. For more information, see [Overview of Office 365 ProPlus](https://go.microsoft.com/fwlink/p/?linkid=846328)**..**

Please add "have" and remove extra period(.)

File: Office365-Admin/manage/centralized-deployment-of-add-ins.md [hunk 5]
```diff
@@ -62,20 +46,41 @@ Centralized Deployment doesn't support the following:
...
-|PowerPoint  <br/> |![checkmark](../media/5277fbec-0a8f-4bd0-b906-d6ddee85a46c.png)           <br/> |![checkmark](../media/5277fbec-0a8f-4bd0-b906-d6ddee85a46c.png)           <br/> |![checkmark](../media/5277fbec-0a8f-4bd0-b906-d6ddee85a46c.png)           <br/> |
-|Outlook  <br/> |![checkmark](../media/5277fbec-0a8f-4bd0-b906-d6ddee85a46c.png)           <br/> |![checkmark](../media/5277fbec-0a8f-4bd0-b906-d6ddee85a46c.png)           <br/> |![checkmark](../media/5277fbec-0a8f-4bd0-b906-d6ddee85a46c.png)           <br/> |
-   
-Office Online is supported for all Office 365 business plans. Office 365 ProPlus deployments are supported for desktop clients only. To use Office 365 ProPlus, a user must have an Office 365 account and been assigned a license. For more information, see [Overview of Office 365 ProPlus](https://go.microsoft.com/fwlink/p/?linkid=846328).
+If you don't want to use the Activations report, you can ask a user to open on their machine an Office application, such as Word, and then choose **File** \> **Account**. Under **Product Information**, you should see **Subscription Product** and **Microsoft Office 365 ProPlus**, as shown in the following image.
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/centralized-deployment-of-add-ins.md]:
```suggestion
If you don't want to use the Activations report, you can ask a user to open **an Office application such as Word on their machine,** and then choose **File** \> **Account**. Under **Product Information**, you should see **Subscription Product** and **Microsoft Office 365 ProPlus**, as shown in the following image.
```

**Comment by rokeates** [comment]
Suggested changes have been made. Thank you for reviewing!

**Pull request merged**